internal/encryption/method/aesgcm/aesgcm.go

```diff
@@ -123,3 +123,8 @@ func (a aesgcm) getGCM(key []byte) (cipher.AEAD, error) {
 	}
 	return gcm, nil
 }
+
+func Is(m method.Method) bool {
```

Comment:
It's used in newly introduced test: https://github.com/opentofu/opentofu/pull/1711/files#diff-89663d138a7de3a7669ab0efd13ac851af80d5f7a6a4179b6e8e250682f5ee1bR40

Currently, it's hard to tell what algorithm is implemented under `method.Method` since there is no trace back to a descriptor. I would expect having `ID()` as a part of `method.Method` interface, but it may break the segregation logic.

`Is` doesn't need to be a part of the descriptor, but I can move it to the `_test.go` so it's only available for testing purposes. I can also just check via reflection, but that's more fragile.